README.md

```diff
@@ -184,50 +184,7 @@ go get github.com/kyma-project/module-manager/operator@latest
 2. As part of the controller's `SetupWithManager()` in the Sample CR [controller implementation](controllers/sample_controller.go), we now have to tell the declarative reconciler how to reconcile the object.
    For this, we need to inject the necessary information about the declarative intention on what to reconcile with `Inject(...)`.
 
-   In its most simple form, the new setup could look like this (assuming all the settings from above were used):
-    
-   <details>
-        <summary>Sample implementation</summary>
-   
-   ```go
-    package controllers
-    import (
-	    "github.com/kyma-project/module-manager/operator/pkg/declarative"
-        metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
-	    "sigs.k8s.io/controller-runtime/pkg/client"
-	    ctrl "sigs.k8s.io/controller-runtime"
-	    "k8s.io/apimachinery/pkg/runtime"
-   
-	    operatorv1alpha1 "github.com/kyma-project/test-operator/operator/api/v1alpha1"
-    )
-   
-    // SampleReconciler reconciles a Sample object
-    type SampleReconciler struct {
-      declarative.ManifestReconciler
-      client.Client
-      Scheme *runtime.Scheme
-    }
-   
-    // SetupWithManager sets up the controller with the Manager.
-    func (r *SampleReconciler) SetupWithManager(mgr ctrl.Manager) error {
-    if err := r.Inject(
-      mgr, &operatorv1alpha1.Sample{},
-      declarative.WithCustomResourceLabels(map[string]string{"sampleKey": "sampleValue"}),
-      declarative.WithResourcesReady(true),
-      declarative.WithFinalizer("sample-finalizer"),
-      declarative.WithDefaultResolver(),
-    ); err != nil {
-      return err
-    }
-    
-    return ctrl.NewControllerManagedBy(mgr).
-	    For(&operatorv1alpha1.Sample{}).
-	    Complete(r)
-    }
-   ```
-   
-   </details>
-   
+   In its most simple form, the new setup could look like this (assuming all the settings from above were used).   
```

Comment:
I think this sentence is very much confusing, since you are not pointing to any examples here. You state it could look like `this`. But where is `this`.

The example you are maybe refer to is at the end of step 3. I think for devs who are going here for the first time, are lost at this point.

Please make sure you are exactly pointing to the example.

Or you could move the example implementation to step 2 and give further details regarding the example afterwards.

Furthermore, I disagree with the wording `most simple form`, since this is not true.
The most simple form would be using the `declarative.WithDefaultResolver()` and not `declarative.WithManifestResolver(<...>)`. Since if the `WithManifestResolver` options gets used, an additional resolver needs to be implemented, which is not the case when using `declarative.WithDefaultResolver()` and thus way simpler for the user.